tests/IntegrationTests.py

```diff
@@ -21,6 +21,7 @@ def percy_snapshot(cls, name=''):
         cls.percy_runner.snapshot(
             name=snapshot_name
         )
+        cls.driver.save_screenshot('/tmp/artifacts/{}.png'.format(name))
```

Comment:
If I understood the purpose correctly, this is to investigate with circile ci docker? once shift to pytest, we can implement a hook to do this automatically when the test case failed.

Reply:
yeah exactly. we can remove this now that the issue has been fixed. Doing it automatically when tests fail would be great though!